e2e/runner_test.go

```diff
@@ -4,7 +4,6 @@ import (
 	"flag"
```

Comment:
This class will be removed in 3, 2, 1...

![dracarys](https://media.giphy.com/media/A7ULitOKxeF20/giphy.gif)